README.md

```diff
@@ -1,7 +1,10 @@
+
 # Mudlet
 
-[![Build Status](https://travis-ci.org/Mudlet/Mudlet.svg?branch=development)](https://travis-ci.org/Mudlet/Mudlet) [![GitHub stars](https://img.shields.io/github/stars/Mudlet/Mudlet.svg)](https://github.com/Mudlet/Mudlet/stargazers)
```

Comment:
This is I think incomplete or wrong, to place all the badges on ONE line they need to be on one in the README.md file whereas the Gitter one was on a separate line (and still is) and the AppVeyor one is incorrect.  The code as it stands in the PR is:

    
    ![https://ci.appveyor.com/api/projects/status/github/Mudlet](https://ci.appveyor.com/api/projects/status/github/Mudlet) [![Build Status](https://travis-ci.org/Mudlet/Mudlet.svg?branch=development)](https://travis-ci.org/Mudlet/Mudlet) [![GitHub stars](https://img.shields.io/github/stars/Mudlet/Mudlet.svg)](https://github.com/Mudlet/Mudlet/stargazers)
    [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/Mudlet/Mudlet?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) 
yielding:

![https://ci.appveyor.com/api/projects/status/github/Mudlet](https://ci.appveyor.com/api/projects/status/github/Mudlet) [![Build Status](https://travis-ci.org/Mudlet/Mudlet.svg?branch=development)](https://travis-ci.org/Mudlet/Mudlet) [![GitHub stars](https://img.shields.io/github/stars/Mudlet/Mudlet.svg)](https://github.com/Mudlet/Mudlet/stargazers)
[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/Mudlet/Mudlet?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) 

whereas it should be 

    [![AppVeyor Build status](https://ci.appveyor.com/api/projects/status/31fqq844br6g6i40/branch/development?svg=true)](https://ci.appveyor.com/project/Mudlet/mudlet/branch/development) [![GitHub Build Status](https://travis-ci.org/Mudlet/Mudlet.svg?branch=development)](https://travis-ci.org/Mudlet/Mudlet) [![GitHub stars](https://img.shields.io/github/stars/Mudlet/Mudlet.svg)](https://github.com/Mudlet/Mudlet/stargazers) [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/Mudlet/Mudlet?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)

yielding:
[![AppVeyor Build status](https://ci.appveyor.com/api/projects/status/31fqq844br6g6i40/branch/development?svg=true)](https://ci.appveyor.com/project/Mudlet/mudlet/branch/development) [![GitHub Build Status](https://travis-ci.org/Mudlet/Mudlet.svg?branch=development)](https://travis-ci.org/Mudlet/Mudlet) [![GitHub stars](https://img.shields.io/github/stars/Mudlet/Mudlet.svg)](https://github.com/Mudlet/Mudlet/stargazers) [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/Mudlet/Mudlet?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)